jekyll/editors.markdown

```diff
@@ -283,11 +283,13 @@ Restart LSP or Sublime Text and `ruby-lsp` will automatically activate when open
 
 ## Zed
 
-[Setting up Ruby LSP](https://github.com/zed-industries/zed/blob/main/docs/src/languages/ruby.md#setting-up-ruby-lsp)
```

Comment:
I've updated this below to point to the docs on zed.dev, rather than the GitHub source.